.gitignore

```diff
@@ -286,3 +286,4 @@ $RECYCLE.BIN/
 doc/
 .idea/
 queen.code-workspace
+storyformats/
```

Comment:
I don't think tweego executable in the repository is a good idea. If so, this change makes no sense. If you want to keep your cpoy that way, you can move this rule to the `.git/info/exclude` file.

Reply:
I'm new to git so I committed wrong I think. The second one was meant only to be kept on my fork until the whole thing was done.

But you are correct I shouldn't have tweego in the repository, forgot to add that to the gitignore.

Reply:
Can you close this and just change the little thing in the first commit. Because I don't how to revert stuff, without fucking stuff up again. So am first going to look more into basic use of github, before I commit, push and whatever not again

Reply:
Hey, I just closed this PR. 

Don't worry about re-balancing stats on things with a PR, instead propose them with an issue. A lot of this stuff is horribly  balanced and my thinking may change from time to time even in my comments.

Don't let it discourage you though, keep trying.